app/components/schools/participants/status_table_row.rb

```diff
@@ -26,6 +26,10 @@ def participant_is_on_a_cip?
         profile.school_cohort.school_chose_cip?
       end
 
+      def path_ids
```

Comment:
Here's another method I added so we're not relying on Rails already knowing the path to build links, it's more explicit and easier to test.